src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(num) {
+      calculator.result += num;
+
+      return calculator;
```

Comment:
```suggestion

```
There is no need to return the calculator from the method add/subtract / etc. We don't need to call them in a chain